Users/services/usersService.js

```diff
@@ -0,0 +1,63 @@
+import { usersDao } from '../models';
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+const TOKEN_KEY = '' + process.env.SECRET_KEY;
+
+const findAllUsers = async () => {
+  return await usersDao.getAllUsers();
+};
+
+const userSignUp = async (
+  email,
+  name,
+  password,
+  account,
+  nickname,
+  phone_number
+) => {
+  const saltRounds = 10;
+  const salt = await bcrypt.genSalt(saltRounds);
+  const hashedPassword = await bcrypt.hash(password, salt);
+
+  const existingEmail = await usersDao.getEmail(email);
+
+  if (existingEmail.length) {
+    const error = new Error('error');
+    error.statusCode = 409;
+    throw error;
+  }
+
+  return await usersDao.createUser(
+    email,
+    name,
+    hashedPassword,
+    account,
+    nickname,
+    phone_number
+  );
+};
```

Comment:
```suggestion

  return await usersDao.createUser(
    email,
    name,
    hashedPassword,
    account,
    nickname,
    phoneNumber
  );
};
```
오 Service와 Dao에서 각자 다른 함수 이름을 사용하셨군요 👍 
적절한 이름들이라 생각됩니다 굿굿

Reply:
감사합니다 소헌님! 😁